cmsl1t/plotting/resolution.py

```diff
@@ -49,30 +53,35 @@ def draw(self, with_fits=False):
         fits = []
         for (pile_up, ), hist in self.plots.flat_items_all():
             if pile_up == bn.Base.everything:
-                hist.linestyle = "dashed"
-                label = "Everything"
+                # hist.linestyle = "dashed"
+                hist.drawstyle = ResolutionPlot.drawstyle
+                label = "All PU"
             elif isinstance(pile_up, int):
-                hist.drawstyle = "EP"
-                label = "~ {:.0f}".format(
-                    self.pileup_bins.get_bin_center(pile_up))
+                hist.drawstyle = ResolutionPlot.drawstyle
+                if self.pileup_bins.get_bin_upper(pile_up) < 500:
+                    label = "{:.0f} \\leq PU < {:.0f}".format(
+                        self.pileup_bins.get_bin_lower(pile_up),
+                        self.pileup_bins.get_bin_upper(pile_up),
+                    )
+                else:
+                    label = "{:.0f} < PU".format(
+                        self.pileup_bins.get_bin_lower(pile_up))
             else:
                 continue
             hist.SetMarkerSize(0.5)
-            hist.SetLineWidth(2)
+            hist.SetLineWidth(1)
             hists.append(hist)
             labels.append(label)
-            # if with_fits:
-            #     fits.append(self.fits.get_bin_contents([pile_up]))
+            if with_fits:
+                fits.append(self.fits.get_bin_contents([pile_up]))
```

Comment:
I think this is a left-over from copying this file from the Efficiency plots, but we don't actually seem to assign to `self.fits`, so I'm pretty sure this would crash.  I might actually be responsible for this, I honestly don't remember anymore, but either way, it does look like this class hasn't been used with any fitting.  Might be worth to comment out again or delete, unless you can be bothered to add the fitting yourself.  As it's a resolution, the fit is simple as a Gaussian (symmetric), or an exponentiallly modified Gaussian (asymmetric), both of which ROOT's TMath has native support for, I believe.